vector/src/main/res/values-fr/strings.xml

```diff
@@ -892,7 +892,7 @@
     <string name="send_suggestion_sent">Merci, la suggestion a bien été envoyée</string>
     <string name="send_suggestion_failed">Échec d’envoi de la suggestion (%s)</string>
     <string name="settings_labs_show_hidden_events_in_timeline">Afficher les évènements cachés dans l’historique</string> <!-- Tchap: Custom string -->
-    <string name="bottom_action_people_x">Conversations privées</string>
+    <string name="bottom_action_people_x">Message directs</string> <!-- Tchap: Custom string -->
```

Comment:
@Claire1817 please update with "Messages Directs"

Reply:
The majuscule is still missing, do you want to have it @giomfo?

Reply:
@Florian14 @giomfo 
I don't know what happen with my git... because i did the changes. 
I will fix all the problems and send you a message when it's ready. 
Sorry